src/client/dbManager.js

```diff
@@ -43,33 +43,45 @@ export default class DbManager {
   }
 
   async syncRights() {
-    const rights = await apiFetchJSON(`db/${this.currentDb}/rights`);
-    this.store.dispatch(setUserRights(rights));
+    const rights = await apiFetchJSONOptional(`db/${this.currentDb}/rights`);
+    if (rights) {
+      this.store.dispatch(setUserRights(rights));
+    }
     return rights;
```

Comment:
Are these bugfixes?

Reply:
Yes it fixes a bug when the selected DB is initialized from local storage but that database does not exist.